client/src/main/java/de/qabel/desktop/ui/actionlog/ActionlogController.java

```diff
@@ -336,13 +338,11 @@ public void selectEmoji() {
         popOver.getStyleClass().add("emojiSelector");
         EmojiSelector emojiSelector = new EmojiSelector();
         emojiSelector.getStylesheets().add(QabelFXMLView.getGlobalStyleSheet());
-        Subscription subscription = emojiSelector.onSelect().subscribe(emoji -> {
-            Platform.runLater(() -> {
-                insert(emoji.getUnicode());
-                textarea.requestFocus();
-                popOver.hide();
-            });
-        });
+        Subscription subscription = emojiSelector.onSelect().subscribe(emoji -> Platform.runLater(() -> {
```

Comment:
instead of the runLater() call: Why don't you implement a scheduler that runs on the platform thread? I this pattern everywhere.